zmqauth/zmq_auth_fill.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+cd $(dirname $0)
+id -u > zmq_auth_gen/id
+docker-compose build || exit $?
+docker-compose up || exit $?
```

Comment:
Can you add a `docker-compose rm` here so it doesn't leave an exited container around?